app/api/openWeatherApi.ts

```
@@ -0,0 +1,263 @@
+    return false;
+  }
+
+  const weatherObj = weather as Record<string, unknown>;
+  return (
+    typeof weatherObj.main === 'string' &&
+    typeof weatherObj.icon === 'string'
+  );
+};
+
+export const getCurrentCityForecastData = async (
+  cityName: string = "Melbourne"
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)